### DISCUSSION TITLE
Alternative data source EVNotify ?

### DISCUSSION THREAD

**Discussion by MattiJarvinen** [tillsteinbach/VWsFriend#58] [Ideas]
It would be interesting to have optionally data pulled from EVNotify logs instead of WeConnect, since VW API lacks a lot, namely positioning & API request limits make data somewhat inaccurate anyways.

That Grafana setup is 🎉and I'd rather keep my own data private instead of someone else's computer.

**Comment by tillsteinbach** [comment]
Dear Matti, thanks for your feedback. I don't know the EVNotify logs. Is that referring to the statistics from your home wallbox? I think I need a little more context.

**Reply by tillsteinbach** [reply]
I had a look into evnotify. Looks nice and it would be definitely doable to collect data from there, but currently the hurdle for me is too big and the usage too low.
I like to keep that you do not need anything extra no hardware, nothing with additional costs right now. You just use what you already have.
If there would be a larger group of interested users I would reconsider, but I would definitely need some crowdfunding for that before as I don’t have any hardware for OBD now.

**Reply by MattiJarvinen** [reply]
Thanks for being open for suggestions and replying so soon. 

I might add something like this later on if I can manage to get some time to use.

**Reply by tillsteinbach** [reply]
Nice! Im of course open for contributions
